cpp/inc/bond/core/transforms.h

```diff
@@ -377,7 +377,7 @@ class RequiredFieldValiadator
     {}
 
 private:
-    void MissingFieldException() const;
+    BOND_NORETURN void MissingFieldException() const;
```

Comment:
Are you sure that is needed specifically for `[[noreturn]]`? It should not impact the definition/implementation.